lstchain/calib/camera/drs4.py

```diff
@@ -35,27 +36,29 @@ def fill_pedestal_event(self, event, nr):
     def _fill_pedestal_event_jit(nr, waveform, expected_pixel_id, first_cap, meanped, numped):
         size4drs = 4096
         roisize = 40
-        for i in range(0, 2):
-            for j in range(0, 7):
+        for i in prange(0, 2):
```

Comment:
Out of curiosity: by how much does this improve the efficiency?
Have you tried without loops using numpy only (in a similar fashion to what I proposed by email some time ago)?

Reply:
Nice.